Community/Tdarr_Plugin_MC93_Migz1FFMPEG.js

```diff
@@ -335,8 +366,73 @@ const plugin = (file, librarySettings, inputs, otherArguments) => {
     response.preset = '-c:v vp8_cuvid';
   }
 
-  response.preset += `${genpts}, -map 0 -c:v hevc_nvenc -cq:v 19 ${bitrateSettings} `
-  + `-spatial_aq:v 1 -rc-lookahead:v 32 -c:a copy -c:s copy -max_muxing_queue_size 9999 ${extraArguments}`;
+  // Determine if GPU is present and which gpu to use based on utilization of each GPU
+  //response.infoLog += 'Running nvidia-smi to see if a GPU is available and selecting the GPU with less utilization\n';
```

Comment:
Commented out `response.infoLog` to avoid build error: `expected to deeply equal`

Reply:
Added this output to test file!